libcst/_nodes/internal.py

```diff
@@ -48,7 +47,10 @@ def add_indent_tokens(self) -> None:
     def add_token(self, value: str) -> None:
         self.tokens.append(value)
 
-    def record_position(self, node: "CSTNode", position: CodeRange) -> None:
+    def before_visit(self, node: "CSTNode") -> None:
+        pass
+
+    def after_leave(self, node: "CSTNode") -> None:
```

Comment:
I'm worried that people will confuse these with the various `visit_X` and `leave_X` functions. Should we name them something like `before_codgen` and `after_codegen` instead?